src/Resources/doc/event_subscribers.md

```diff
@@ -31,3 +31,13 @@ csa_guzzle:
                 logger: false
                 my_subscriber: false # Note the use of the alias defined earlier in the service definition.
 ```
+
+When registering your own clients with the bundle, you can explicitly list all
+enabled subscribers. In that case any other subscriber will be disabled for that
+client:
```

Comment:
We should mention that the subscribers key is a comma-delimited list of subscriber names.